FluentTerminal.App/Views/SettingsPages/TerminalSettings.xaml

```diff
@@ -57,6 +57,15 @@
                     Minimum="0"
                     StepFrequency="0.01"
                     Value="{x:Bind ViewModel.BackgroundOpacity, Mode=TwoWay}" />
+                <Slider
+                    Width="200"
+                    Margin="0,0,0,24"
+                    HorizontalAlignment="Left"
+                    Header="Padding"
+                    Maximum="100"
```

Comment:
Although it is very subjective, I don't think anyone would want a padding of `100px`. Personally I would set this to something in the range of `30` (perhaps somewhat low) to `50` (already very high).

Reply:
I agree. I think 30 is enough.